python_modules/dagster/dagster_tests/cli_tests/command_tests/test_cli_commands.py

```diff
@@ -157,6 +146,11 @@ def error_config(_):
         raise Exception("womp womp")
 
     return {
+        "partitioned_pipeline_partitions": PartitionSetDefinition(
+            name="partitioned_pipeline_partitions",
+            pipeline_name="partitioned_pipeline",
+            partition_fn=lambda: string.digits,
+        ),
         "baz_partitions": PartitionSetDefinition(
             name="baz_partitions",
             pipeline_name="baz",
```

Comment:
hrm what's the deal here - ideally there should be no `PartitionSetDefinitions` after this

Reply:
This one is grouped with all the others that were already there and I'm taking aim at all PartitionsSetDefinition in the next PR in this stack